Runtime/Tiles/RuleTile/RuleTile.cs

```diff
@@ -76,7 +76,10 @@ public class TilingRuleOutput
             /// <summary>
             /// The output Animation Speed for this Rule.
             /// </summary>
-            public float m_AnimationSpeed = 1f;
+            public float m_MinAnimationSpeed = 1f;
```

Comment:
`FormerlySerializedAs` may nice for `m_MinAnimationSpeed` and `m_MaxAnimationSpeed`.

Reply:
Good idea! Wasn't aware that is a thing.